Libplanet.Tests/Action/AccountStateDeltaTest.cs

```diff
@@ -0,0 +1,331 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Numerics;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Libplanet.Blockchain;
+using Libplanet.Crypto;
+using Libplanet.Store;
+using Libplanet.Store.Trie;
+using Libplanet.Tests.Blockchain;
+using Libplanet.Tests.Common.Action;
+using Libplanet.Tx;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Libplanet.Tests.Action
+{
+    public abstract class AccountStateDeltaTest
+    {
+        protected readonly PrivateKey[] _keys;
+        protected readonly Address[] _addr;
```

Comment:
How about renaming this variable name as plural like `_addresses` as other variables do? Is it too long? 🤔

Reply:
First of all, these names were not named in this pull request.  The file was renamed and slightly modified (class name, etc).

I guess these names were named such short with intention so that they look like prefixes rather than their own names.  If you look into their usages, they indeed are always used with indexers…